report/3.Theory/ObjectRecognition.tex

```diff
@@ -18,9 +18,125 @@ \subsection{Neural Networks}
 One of the reasons neural networks are especially interesting in this case, is its ability to recognize patterns.
 Pattern recognition is a fundamental part of object recognition making neural networks an obvious option.
 
-\subsubsection{Convolutional Neural Network}
+\subsubsection{The structure of a neural network}
+
+A typical neural network consists of an input layer, an output layer and one or multiple hidden layers in between.
+It is the task of the hidden layers to transform the input from the input layer into the desired output onto the output layer.
+The layers consists of one or more neurons, where in a normal fully connected neural network all neurons in a layer is connected to all neurons in the previous layer.
+
+All connections in a neural network has a weight associated to it, which acts as a simple multiplier for the output from the previous neuron. Apart from the weights all neurons, except for input neurons, are assigned a bias, which is added to the sum of all inputs to the neuron. The bias is sometimes thought of as a sort of threshold indicating whether a neuron should fire.
+
+\figur{0.7}{images/NN.png}{Illustration of a simple neural network with 2 neurons in each layer\cite{NeuralNetworkImage}.}{fig:nnimage}
+
+Before passing the result of a neuron on, the result is first transformed by an activation function. The activation function is required in order for the neural network to learn to solve a complex non-linear model. This will be further discussed in \ref{training-a-neural-network} about training.
```

Comment:
```suggestion
The input, after being modified by the weights, is sent through an activation function.
The activation function is required in order for the neural network to learn to solve a complex non-linear model.
This will be further discussed in \ref{training-a-neural-network} about training.
```